test/contrib/k8s_job_test.py

```diff
@@ -62,22 +76,38 @@ class FailJob(KubernetesJobTask):
         }]
     }
 
+    def signal_complete(self):
+        with self.output().open('w') as output:
+            output.write('')
+
+    def output(self):
+        """This will not be written to."""
+        target = os.path.join("/tmp", "failjob")
+        return luigi.LocalTarget(target)
+
+
 class TestK8STask(unittest.TestCase):
 
     def test_success_job(self):
         success = luigi.run(["SuccessJob", "--local-scheduler"])
         self.assertTrue(success)
 
+        # a file should have been touched on job completion
```

Comment:
Please remove this form this test, and test in a separate test case (SignalCompleteJob).